modules/ROOT/pages/6.5-release-notes.adoc

```diff
@@ -73,6 +73,12 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 {productname} 6.5 also includes the following improvements:
 
+=== Screen readers now announce instructions for resizing the editor using arrow keys, when the resize handle is focused.
+//#TINY-9793
+
+When using voice assistant software in previous versions of {productname}, focusing on the editor resize handle would not announce any instruction.
+
+In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.
```

Comment:
```suggestion
In {productname} 6.5, instructions for adjusting the size of the editor using arrow keys are now announced upon focusing the editor resize handle, while using voice assistant software.

NOTE: In {productname} 6.5 Enterprise, this announcement is only available in English. Additional translations will be added in a future release.
```
There should be a known issue about the translations, both here and under "Known issues". Forgot about this until today, sorry.